src/zenml/integrations/airflow/__init__.py

```diff
@@ -30,9 +28,7 @@ class AirflowIntegration(Integration):
     """Definition of Airflow Integration for ZenML."""
 
     NAME = AIRFLOW
-    # remove pendulum version requirement once Airflow supports
-    # pendulum>-3.0.0
-    REQUIREMENTS = ["apache-airflow~=2.4.0", "pendulum<3.0.0"]
+    REQUIREMENTS = []
```

Comment:
Hm, this is interesting one: how Airflow integration can have no reqs at all?

Reply:
Airflow still has a dependency on `sqlalchemy` V1 and this conflicts with this entire PR as we have to migrate to `sqlalchemy` V2. However, we managed to figure out a way where we can still run pipelines on Airflow by keeping the Airflow and ZenML installations separated.

In a very basic explanation, ZenML can create the corresponding DAG files and Airflow can run the generated files in a separate environment.